runtime/binding-asyncapi/src/main/java/io/aklivity/zilla/runtime/binding/asyncapi/internal/config/composite/AsyncapiCompositeGenerator.java

```diff
@@ -126,7 +126,8 @@ public final AsyncapiCompositeConfig generate(
                         : specification.servers;
                 final AsyncapiView asyncapi = AsyncapiView.of(tagIndex++, label, parser.parse(payload), configs);
 
-                unresolved.addAll(asyncapi.resolver.unresolved());
+                unresolved.addAll(asyncapi.resolver.unresolvedRefs());
```

Comment:
Perhaps don't expose `resolver` as `public` and instead add a method to `AsyncapiView` called `unresolvedRefs()` that delegates to `resolver.unresolvedRefs()`.